### DISCUSSION TITLE
Repo of the Conductor UI

### DISCUSSION THREAD

**Discussion by meggarr** [Netflix/conductor#3793] [Q&A]
Hi theres,

Does anyone know that is the Conductor UI in the main repo ( https://github.com/Netflix/conductor ) or in the dedicated conductor-ui repo ( https://github.com/Netflix/conductor-ui ) ?

Thanks in advance,

**Comment by meggarr** [comment]
Does anyone know this ?

**Reply by JCHacking** [reply]
I use the one from this repository, you have it in the ui section.

The other I don't know if it gives any extra feature, in conductor (The backend) for example this repository and then the conductor-community repository that has different extra features that the community has added.

**Reply by JCHacking** [reply]
Hi @meggarr 
I have been researching, the conductor-ui repository is the main one. The one in this repository is in maintenance mode and only receives security updates.
